pkg/queue/cluster_queue_test.go

```diff
@@ -179,7 +180,9 @@ func Test_PushOrUpdate(t *testing.T) {
 
 func Test_Pop(t *testing.T) {
 	now := time.Now()
-	cq := newClusterQueueImpl(defaultOrdering, testingclock.NewFakeClock(now))
+	fakeClock := testingclock.NewFakeClock(now)
+
+	cq := newClusterQueueImpl(defaultOrdering, fakeClock)
```

Comment:
I was not sure the benefit such refactoring.
What is the reason that we need to additional malloc to new variable?

This seems just increase memory allocation.